src/BloomExe/Book/Book.cs

```diff
@@ -679,6 +679,96 @@ public void BringBookUpToDate(IProgress progress)
 			}
 		}
 
+		class GuidAndPath
+		{
```

Comment:
I don't think the difference is important for objects which are small and immutable like these. See https://msdn.microsoft.com/en-us/library/ms173109.aspx for the difference...basically variables of type class are references, assigning one to another makes them refer to the same instance, while structs are value types, and assignment copies the content. This usually produces surprising results if a struct is mutable. In this case a struct would be possible; I don't usually bother with them unless some particular optimization is achieved by using one.